assets/base.css

```diff
@@ -929,20 +929,20 @@ summary::-webkit-details-marker {
   display: flex;
   flex-wrap: wrap;
   margin-bottom: 2rem;
-  margin-left: -0.5rem;
+  margin-left: calc(-1 * var(--grid-mobile-horizontal-spacing));;
```

Comment:
TIL you need to use Calc to achieve this.

Reply:
Yeah. I couldn't get it to work otherwise. Maybe there's another way, but negatives were not working for me without `calc`